src/mono/nuget/Microsoft.NET.Runtime.WebAssembly.Sdk/Microsoft.NET.Runtime.WebAssembly.Sdk.pkgproj

```diff
@@ -0,0 +1,18 @@
+<Project DefaultTargets="Build">
```

Comment:
Do we really have to call it SDK? I don't think we want to make any guarantees about this package so the name should reflect that.

Reply:
`WebAssembly.Internal`? `WebAssembly.Internal.Pack`?

Reply:
Few questions we should perhaps answer first
- Is this WebAssembly or Browser-WebAssembly target (e.g. will node-wasm use this package)?
- What are we going to put inside (e.g will other tasks be included (e.g. debugger pieces) or ICU tooling?
- Should the naming pattern be same for other workloads with the need for publish/build tasks?

Reply:
It's going to be something we ship as an SDK pack and having `Internal` in the name would appear suspicious. We should certainly try and steer clear of that.